app/javascript/dashboard/i18n/locale/en/sla.json

```diff
@@ -0,0 +1,63 @@
+{
+  "SLA": {
+    "HEADER": "SLA",
+    "HEADER_BTN_TXT": "Add SLA",
+    "LOADING": "Fetching SLAs",
+    "SEARCH_404": "There are no items matching this query",
+    "SIDEBAR_TXT": "<p><b>SLA</b> <p>Think of Service Level Agreements (SLAs) like friendly promises between a service provider and a customer.</p> <p> These promises set clear expectations for things like how quickly the team will respond to issues, making sure you always get a reliable and top-notch experience!</p>",
+    "LIST": {
+      "404": "There are no SLAs available in this account.",
+      "TITLE": "Manage SLA",
+      "DESC": "SLAs: Friendly promises for great service!",
+      "TABLE_HEADER": ["Name", "Description", "FRT", "NRT", "RT", "Business Hours"]
+    },
+    "FORM": {
+      "NAME": {
+        "LABEL": "SLA Name",
+        "PLACEHOLDER": "SLA Name",
+        "REQUIRED_ERROR": "SLA name is required",
+        "MINIMUM_LENGTH_ERROR": "Minimum length 2 is required",
+        "VALID_ERROR": "Only Alphabets, Numbers, Hyphen and Underscore are allowed"
+      },
+      "DESCRIPTION": {
+        "LABEL": "Description",
+        "PLACEHOLDER": "SLA for premium customers"
+      },
+      "FIRST_RESPONSE_TIME": {
+        "LABEL": "First Response Time(Seconds)",
+        "PLACEHOLDER": "300 for 5 minutes"
+      },
+      "NEXT_RESPONSE_TIME": {
+        "LABEL": "Next Response Time(Seconds)",
+        "PLACEHOLDER": "600 for 10 minutes"
+      },
+      "RESOLUTION_TIME": {
+        "LABEL": "Resolution Time(Seconds)",
+        "PLACEHOLDER": "86400 for 1 day"
```

Comment:
Let us convert this internally and take input properly from the customer. 

Instead of 600 in seconds fields, let us give input with a dropdown and convert the value to seconds. 

<img width="791" alt="Screenshot 2024-02-15 at 6 44 42 PM" src="https://github.com/chatwoot/chatwoot/assets/2246121/2d9e1246-27f6-412e-b0a0-c1cf78acd629">

Input number + Dropdown with enum (seconds, minutes, hours, days)

Reply:
Will address this in a separate PR
https://github.com/chatwoot/chatwoot/pull/8974